source/DataAccess/DADGenerator.ts

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+import IntegrationConfigFactory from '../IntegrationConfigFactory';
+import { IIntegrationConfig, IntegrationType } from '../IIntegrationConfig';
+import { writeFile } from 'fs';
+
+export default class DataAccessDoc {
+
+    private _integrationType;
+    private _queries;
+    private _header: string[] = [];
+    private _footer: string[] = [];
+
+    constructor(integrationType: IntegrationType) {
+        this._integrationType = integrationType.toString();
+        const icf = new IntegrationConfigFactory();
+        const cfg: IIntegrationConfig = icf.create(this._integrationType);
+        this._queries = cfg.queries;
+        this._header.push(`# Data Access Requirements: ${this._integrationType}`);
+        this._header.push(`DIA requires access to the following tables/fields in the ${this._integrationType} integration type.`);
+    }
+
+    public create(): void {
+        const path = './docs/DataAccess/' + this._integrationType + '.md';
+        const n = '\n';
+        const dn = n + n;
+        const data = this._header.join(n) + dn + this._createTable().join(n) + dn + this._footer.join(n);
```

Comment:
I'd suggest maybe renaming 'n' to something like 'newLine'. You could also get rid of 'dn' and use newLine.repeat(2) - or - you could do .join(n.repeat(3)) when you're doing .join(n) + dn. It's all a little more wordy but gets the message across better I think.